packages/lit-html/package.json

```diff
@@ -460,9 +460,7 @@
         "tsconfig.compiled-tests.json"
       ],
       "output": [
-        "compiled/**/*.{js,js.map,d.ts,d.ts.map}",
-        "tsconfig.tsbuildinfo",
```

Comment:
How did you notice this?

Reply:
I saw that on repeat builds wireit was detecting output files modified. I checked the outputs and found the compile build doesn't generate the tsbuildinfo files.